src/number_theory/padics/padic_norm.lean

```diff
@@ -65,59 +62,51 @@ else
     exact_mod_cast nat.zero_le _
   end
 
-/-- The p-adic norm of 0 is 0. -/
+/-- The `p`-adic norm of `0` is `0`. -/
 @[simp] protected lemma zero : padic_norm p 0 = 0 := by simp [padic_norm]
 
-/-- The p-adic norm of 1 is 1. -/
+/-- The `p`-adic norm of `1` is `1`. -/
 @[simp] protected lemma one : padic_norm p 1 = 1 := by simp [padic_norm]
 
-/--
-The p-adic norm of `p` is `1/p` if `p > 1`.
+/-- The `p`-adic norm of `p` is `1/p` if `p > 1`.
 
-See also `padic_norm.padic_norm_p_of_prime` for a version that assumes `p` is prime.
--/
-lemma padic_norm_p {p : ℕ} (hp : 1 < p) : padic_norm p p = 1 / p :=
+See also `padic_norm.padic_norm_p_of_prime` for a version assuming `p` is prime. -/
+lemma padic_norm_p (hp : 1 < p) : padic_norm p p = 1 / p :=
 by simp [padic_norm, (pos_of_gt hp).ne', padic_val_nat.self hp]
 
-/--
-The p-adic norm of `p` is `1/p` if `p` is prime.
+/-- The `p`-adic norm of `p` is `1/p` if `p` is prime.
```

Comment:
```suggestion
/-- The `p`-adic norm of `p` is `1 / p` if `p` is prime.
```
Is there a reason not to use `p⁻¹` instead?

Reply:
I don't actually know.

Reply:
You can just try it.

Reply:
Well, the fact that `⁻¹` exists suggest it should be used, making it better than `1 / p`. But I totally agree one should also change the Lean code, and I don't know if this will break something or not. If it's too much work don't worry.